service/instance.py

```diff
@@ -2032,25 +2037,25 @@ def run_instance_action(user, identity, instance_id, action_type, action_params)
     elif 'suspend' == action_type:
         result_obj = suspend_instance(esh_driver, esh_instance,
                                       provider_uuid, identity_uuid,
-                                      user)
+                                      user, reclaim_ip)
     elif 'shelve' == action_type:
         result_obj = shelve_instance(esh_driver, esh_instance,
                                      provider_uuid, identity_uuid,
-                                     user)
+                                     user, reclaim_ip)
     elif 'unshelve' == action_type:
         result_obj = unshelve_instance(esh_driver, esh_instance,
                                        provider_uuid, identity_uuid,
                                        user)
     elif 'shelve_offload' == action_type:
-        result_obj = offload_instance(esh_driver, esh_instance)
+        result_obj = offload_instance(esh_driver, esh_instance, reclaim_ip)
```

Comment:
Problem.

Reply:
See how it's called here:

https://github.com/cyverse/atmosphere/blob/7778e1e7ae8e781d4689af14110b4285247a4976/service/monitoring.py#L178

Also should we add `reclaim_ip` to this occurrence?

Reply:
@julianpistorius Yes, ill address those shortly thanks for bringing it to my attn